src/app/browser/qgsinbuiltdataitemproviders.cpp

```diff
@@ -174,6 +174,51 @@ void QgsAppDirectoryItemGuiProvider::populateContextMenu( QgsDataItem *item, QMe
   } );
   menu->addAction( hideAction );
 
+  QMenu *hiddenMenu = new QMenu( tr( "Hidden items" ), menu );
+  int count = 0;
+  const QStringList hiddenPathList = settings.value( QStringLiteral( "/browser/hiddenPaths" ) ).toStringList();
+  for ( const QString &path : hiddenPathList )
+  {
+    QAction *action = new QAction( path, hiddenMenu );
```

Comment:
```suggestion
    QAction *action = new QAction( QDir::toNativeSeperators( path ), hiddenMenu );
```

Reply:
In this case we probably should use native separators also in the Options dialog to have consistent naming?

Reply:
That would be a good move -- in general we should always format paths using this for display purposes only, so that Windows users get the expected results and not the "raw" looking forward slash versions